lib/kamal/commands/registry.rb

```diff
@@ -2,7 +2,7 @@ class Kamal::Commands::Registry < Kamal::Commands::Base
   delegate :registry, to: :config
 
   def login
-    docker :login, registry["server"], "-u", sensitive(lookup("username")), "-p", sensitive(lookup("password"))
+    docker :login, registry["server"], "-u", sensitive(lookup("username")), "-p", sensitive("'#{lookup("password")}'")
```

Comment:
This wouldn't work if the password contained a `'`.

We could use [String.shellescape](https://docs.ruby-lang.org/en/3.2/Shellwords.html#method-i-shellescape) instead. Even though it's unlikely to contain anything that need it, we should also escape the username.